### DISCUSSION TITLE
Live terminal synced between users

### DISCUSSION THREAD

**Discussion by AndyIsHereBoi** [coder/code-server#5038] [Q&A] [answered]
Is it possile to have a terimnal synced between users while it being live updating for both of them? This is my only problem where one user cannot view another users terminal while working on the same project.

**Comment by jsjoeio** [comment, marked as answer]
That's a great question! So I know with tools like Live Share, you can share terminals and I believe it syncs. Have you tried using [Duckly](https://coder.com/docs/code-server/latest/collaboration#code-sharing-using-duckly)? It says it has terminal sharing but it's been a while since I've used it.

**Reply by AndyIsHereBoi** [reply]
ill try it out

**Comment by AndyIsHereBoi** [comment]
That seems to work great

**Reply by jsjoeio** [reply]
Hooray! Glad to hear it
